src/storage/mod.rs

```diff
@@ -49,3 +49,13 @@ pub(crate) fn save(path: &PathBuf, data: String) -> Result<(), Error> {
     .write_all(data.as_bytes())
     .map_err(|e| Error::IOError(format!("fail to write file: {}", e)))
 }
+
+pub(crate) fn remove_dir(path: &PathBuf) -> Result<(), Error> {
```

Comment:
this is wrong because method delete parent folder, not provided one

Reply:
corrected to path.as_path() instead of path.parent()